### PR TITLE
Add new  `onlyShowCountdownForRealtime` Option

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**

Blocked by #1042 

Supports better countdown clock behavior where it only shows up if there's realtime data! Way better.

**PR Checklist:**
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by AdrianaCeric — APPROVED** [review]
Great change!

File: lib/components/viewers/stop-time-cell.tsx [hunk 1]
```diff
@@ -109,7 +114,10 @@ const StopTimeCell = ({
           status: getTripStatus(realtime, stopTime.departureDelay, 30) as status
         })}
       >
-        {showCountdown ? (
+        {/* Not the cleanest boolean, but makes it very clear what we're doing
+        in all cases. */}
+        {(onlyShowCountdownForRealtime === true && realtime) ||
+        (onlyShowCountdownForRealtime === false && showCountdown) ? (
```
Comment (amy-corson-ibigroup) [inline lib/components/viewers/stop-time-cell.tsx:120]: Can this not be 
```suggestion
        {(onlyShowCountdownForRealtime && realtime) ||
        (!onlyShowCountdownForRealtime && showCountdown) ? (
```

**Review by amy-corson-ibigroup — APPROVED** [review]
I like this change! Just a question about the boolean but otherwise looks good.

File: lib/components/viewers/stop-time-cell.tsx (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/components/viewers/stop-time-cell.tsx:120]: The explicit `true` helps us avoid false/null/undefined confusion

**Pull request merged**